app/controllers/shipit/merge_status_controller.rb

```diff
@@ -60,7 +60,7 @@ def stack
       @stack ||= if params[:stack_id]
         Stack.from_param!(params[:stack_id])
       else
-        scope = Stack.order(id: :asc).where(
+        scope = Stack.order(merge_queue_enabled: :desc, id: :asc).where(
```

Comment:
Is orderring on boolean standard SQL or is it a MySQLism ?

Reply:
I suppose CI will tell us.

Reply:
Good question. Reading about this, SQLite and MySQL store bools as ints, but Postgres doesn't, and when sorting ascending [true comes first](https://code.djangoproject.com/ticket/19726) (this bug is old, but looks like the facts are still correct. I havent tested myself). We chatted IRL and decided to do the sorting in Ruby and try the branch in prod - this endpoint is hit a lot and needs to be performant, but at Shopify the number of stacks configured with the same branch is very small, so this should be fine.

Reply:
Hmm. CI _did_ pass 🤔